### DISCUSSION TITLE
第5节课笔记(2班)

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/Tutorial#322] [笔记]
## 2班第五节课笔记交这里哟~
* 链接形式提交
* 也记得在信息收集文档对应作业处填写链接

**Comment by xrxu** [comment]
第五课笔记: https://github.com/xrxu/Docu/blob/master/lmdeploy/lmdeploy.md

**Comment by DarthThomas** [comment]
[第五课笔记](https://share-tianhe.notion.site/929a6a5a947c452b93be3a6fc888e250?pvs=4)

**Comment by xulinui** [comment]
https://h90ag9106t.feishu.cn/docx/X8eudSxnBowylDxETKcc75JVnyb

**Comment by yangff0016** [comment]
[第五节课笔记](https://blog.csdn.net/weixin_42837310/article/details/135633026)

**Comment by CoderJackZhu** [comment]
[第五节课笔记](https://zhuanlan.zhihu.com/p/677903943?)

**Comment by ZixinxinWang** [comment]
https://blog.csdn.net/Wzxdecsdn/article/details/135623580

**Comment by pranerd** [comment]
[第五课笔记](https://blog.csdn.net/gaxzj/article/details/135707056?spm=1001.2014.3001.5502)

**Comment by RyanWu31** [comment]
https://github.com/RyanWu31/tutorial_notebook/blob/main/README.md

**Comment by songyuc** [comment]
[《(5)LMDeploy 大模型量化部署实践》学习笔记](https://blog.csdn.net/songyuc/article/details/135738212)

**Comment by YanxingLiu** [comment]
第五节课笔记 https://s5eixc6gba.feishu.cn/docx/IDqndoXiqoJZ30x4fiSceoTunOe?from=from_copylink

**Comment by zyy0831** [comment]
第五节课笔记 https://www.yuque.com/yuanyang-xb8hd/eeag2y/focfo9v27ccg2ssn

**Comment by Mr-Vincenyang** [comment]
[第五节课笔记](https://zhuanlan.zhihu.com/p/679290233)

**Comment by PrinceRay7** [comment]
第五节课笔记
https://blog.csdn.net/Prince_Ray/article/details/135833924?spm=1001.2014.3001.5501

**Comment by kishiokon** [comment]
[LMDeploy 大模型量化部署实践](https://www.bilibili.com/read/cv30273146/?jump_opus=1)
